pkg/syncer/util.go

```diff
@@ -56,6 +56,31 @@ func getBoundPVs(ctx context.Context, metadataSyncer *metadataSyncInformer) ([]*
 	return boundPVs, nil
 }
 
+// getInlineMigratedVolumesInfo is a helper function for retrieving  inline PV information from Pods
+func getInlineMigratedVolumesInfo(ctx context.Context, metadataSyncer *metadataSyncInformer) (map[string]string, error) {
+	log := logger.GetLogger(ctx)
+	inlineVolumes := make(map[string]string)
+	// Get all Pods from kubernetes
```

Comment:
if `metadataSyncer.configInfo.Cfg.FeatureStates.CSIMigration` is disabled, we can return empty map right away without traversing Pods.